tests/test_evm.py

```diff
@@ -1,13 +1,17 @@
 import json
-import os.path
+import os
 
-import pytest
 from pyrevm import EVM, Env, BlockEnv, AccountInfo
 
+from tests.utils import load_contract_bin, encode_uint, encode_address
+import pytest
+
+
 address = "0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045"  # vitalik.eth
 address2 = "0xbBbBBBBbbBBBbbbBbbBbbbbBBbBbbbbBbBbbBBbB"
 
-fork_url = "https://mainnet.infura.io/v3/c60b0bb42f8a4c6481ecd229eddaca27"
+# use your own key during development to avoid rate limiting the CI job
+fork_url = os.getenv("FORK_URL") or "https://mainnet.infura.io/v3/c60b0bb42f8a4c6481ecd229eddaca27"
```

Comment:
whose key is this and why is it in the codebase?

Reply:
This key was here from the beginning, probably from @gakonst - removing it will break unit tests for PRs

Reply:
lol 383 instances